### DISCUSSION TITLE
Unique Number of Authors in Author_Insititute array

### DISCUSSION THREAD

**Discussion by karam86** [snap-stanford/ogb#203] [MAG240M-LSC]
I just want to verify that we have only 38.67M unique Authors in author_insitute edges array. While in author_paper array, we have 122M unique authors. Does that mean, for rest of the Authors, we do not have affiliation?.. Here is the snapshot of my code.

![image](https://user-images.githubusercontent.com/36695037/117869885-fa6b6180-b2b8-11eb-9f46-139d607975a1.png)

**Comment by weihua916** [comment]
Hi! Great point. Yes, this is indeed the case, as real-world data is incomplete.
